### PR TITLE
Fix ios debugger for projects created with framework version lower than 1.2.0…

### PR DESCRIPTION
**Pull request by Fatme**
If the project is created with framework version lower than 1.2, `tns debug ios` command fails with the following error: "Command failed: /bin/sh -c open -a '~/.npm/tns-ios/1.1.2/package/WebInspectorUI/NativeScript Inspector.app' --args '~/.npm/tns-ios/1.1.2/package/WebInspectorUI/Safari/Main.html' 'newAppRelease'
FSPathMakeRef(~/.npm/tns-ios/1.1.2/package/WebInspectorUI/NativeScript Inspector.app) failed with error -43."

### CODE REVIEW

File: lib/services/ios-debug-service.ts [hunk 1]
```diff
@@ -205,9 +208,20 @@ class IOSDebugService implements IDebugService {
...
-            let inspectorSourceLocation = path.join(inspectorPath, "Safari/Main.html");
-            let cmd = `open -a '${inspectorApplicationPath}' --args '${inspectorSourceLocation}' '${this.$projectData.projectName}'`;
+            let cmd: string = null;
+            
+            this.$projectDataService.initialize(this.$projectData.projectDir);
+            let platformData = this.$platformsData.getPlatformData(this.platform);
+            let frameworkVersion = this.$projectDataService.getValue(platformData.frameworkPackageName).wait().version;
+            if(semver.lt(frameworkVersion, "1.2.0")) {
+                let safariPath = path.join(inspectorPath, "Safari/Main.html");
+                cmd = "open -a Safari " + safariPath;
```
Comment (teobugslayer) [inline lib/services/ios-debug-service.ts]: is it possible safariPath to have spaces in it? Should we escape it?

File: lib/services/ios-debug-service.ts [hunk 2]
```diff
@@ -205,9 +208,20 @@ class IOSDebugService implements IDebugService {
...
+            
+            this.$projectDataService.initialize(this.$projectData.projectDir);
+            let platformData = this.$platformsData.getPlatformData(this.platform);
+            let frameworkVersion = this.$projectDataService.getValue(platformData.frameworkPackageName).wait().version;
+            if(semver.lt(frameworkVersion, "1.2.0")) {
+                let safariPath = path.join(inspectorPath, "Safari/Main.html");
+                cmd = "open -a Safari " + safariPath;                
+            } else {
+                let inspectorApplicationPath = path.join(inspectorPath, "NativeScript Inspector.app");
+                let inspectorSourceLocation = path.join(inspectorPath, "Safari/Main.html");
```
Comment (teobugslayer) [inline lib/services/ios-debug-service.ts]: this path is calculated both in if and else, consider moving it out and calculating it once.

File: lib/services/ios-debug-service.ts (hunk 1 above)
Comment (Fatme) [inline lib/services/ios-debug-service.ts]: It is possible only when the user changes the default npm cache path :smile_cat:  I'll escape it.

**Comment by teobugslayer** [comment]
:+1:

**Comment by dtopuzov** [comment]
run ci

**Pull request merged**